install/roles/qpc_server/tasks/main.yml

```diff
@@ -10,6 +10,26 @@
     server_port: "{{ ansible_env['SERVER_PORT'] | default('9443') }}"
   when: server_port is not defined
 
+- name: Set open port
```

Comment:
since we added the ability to set this flag we need to document it in the `install.sh` usage